cpp/src/arrow/adapters/orc/adapter.h

```diff
@@ -102,6 +102,26 @@ class ARROW_EXPORT ORCFileReader {
   Status ReadStripe(int64_t stripe, const std::vector<int>& include_indices,
                     std::shared_ptr<RecordBatch>* out);
 
+  /// \brief Seek to designated row. Invoke NextStripeReader() after seek
+  ///        will return stripe reader starting from designed row.
+  ///
+  /// \param[in] rowNumber the rows number to seek
+  Status Seek(int64_t rowNumber);
+
+  /// \brief Get stripe reader
+  ///
+  /// \param[in] readSize the record batch reader default read size
+  /// \param[out] out the returned stripe reader
+  Status NextStripeReader(int64_t readSize, std::shared_ptr<RecordBatchReader>* out);
```

Comment:
it is not clear to me how these methods relate to ReadStrip above, could you document intended usage of the class?